src/main/java/org/tron/core/db2/core/SnapshotManager.java

```diff
@@ -187,6 +187,7 @@ public void shutdown() {
         TimeUnit.MILLISECONDS.sleep(10);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Assign this magic number 10 to a well-named constant, and use the constant instead. ([squid:S109](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS109))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1639&issues=AWalU-zulHK86Hco_1ps&open=AWalU-zulHK86Hco_1ps)

[ISSUE_KEY]: AWalU-zulHK86Hco_1ps